tensorflow/java/src/main/java/org/tensorflow/Operand.java

```diff
@@ -22,19 +22,19 @@
  *
  * <pre>{@code
  * // The "decodeJpeg" operation can be used as an operand to the "cast" operation
- * Operand decodeJpeg = ops.image().decodeJpeg(...);
+ * Operand<TFUint8> decodeJpeg = ops.image().decodeJpeg(...);
  * ops.math().cast(decodeJpeg, DataType.FLOAT);
  *
  * // The output "y" of the "unique" operation can be used as an operand to the "cast" operation
- * Output y = ops.array().unique(...).y();
- * ops.math().cast(y, DataType.FLOAT);
+ * Output<TFInt32> y = ops.array().unique(...).y();
+ * ops.math().cast(y, TFFloat);
```

Comment:
This line doesn't change, I think.
(Or at least let's leave it for now, since the code generation isn't even in place yet, so I'm not quite sure how it will evolve)